test/DynamoCoreTests/AnalyticsTests.cs

```diff
@@ -79,55 +75,64 @@ public override void Cleanup()
         protected virtual void VerifyEventTracking(Times times)
         {
             Analytics.TrackEvent(Actions.New, Categories.NodeOperations, "New Node", 5);
+            Thread.Sleep(100);
             clientMoq.Verify(c => c.TrackEvent(Actions.New, Categories.NodeOperations, "New Node", 5), times);
 
             Analytics.TrackScreenView("TestScreen");
+            Thread.Sleep(100);
             clientMoq.Verify(c => c.TrackScreenView("TestScreen"), times);
 
             Analytics.TrackActivityStatus("User");
+            Thread.Sleep(100);
             clientMoq.Verify(c => c.TrackActivityStatus("User"), times);
 
             TestAnalytics.TrackException<InvalidOperationException>(false);
+            Thread.Sleep(100);
             clientMoq.Verify(c => c.TrackException(It.IsAny<InvalidOperationException>(), false), times);
 
             var time = TimeSpan.FromMinutes(3);
             var variable = "TimeVariable";
             var description = "Some description";
             Analytics.TrackTimedEvent(Categories.Stability, variable, time, description);
+            Thread.Sleep(100);
             clientMoq.Verify(c => c.TrackTimedEvent(Categories.Stability, variable, time, description), times);
 
-            using (var x = Analytics.CreateTimedEvent(Categories.Performance, variable, description))
+            using (var x = Analytics.CreateTaskTimedEvent(Categories.Performance, variable, description).Result)
             {
-                clientMoq.Verify(c => c.CreateTimedEvent(Categories.Performance, variable, description, null), times);
+                clientMoq.Verify(c => c.CreateTaskTimedEvent(Categories.Performance, variable, description, null), times);
             }
 
-            var e = Analytics.TrackCommandEvent("TestCommand");
-            clientMoq.Verify(c => c.CreateCommandEvent("TestCommand", "", null), times);
-
-            Task task = Analytics.TrackTaskCommandEvent("TestCommand", "TestCommand description", null, new Dictionary<string, object>() { } );
-            task.Wait();
+            var e = Analytics.TrackTaskCommandEvent("TestCommand").Result;
+            clientMoq.Verify(c => c.CreateTaskCommandEvent("TestCommand", "", null, null), times);
 
-            clientMoq.Verify(c => c.CreateCommandEvent("TestCommand", "", null), times);
+            e = Analytics.TrackTaskCommandEvent("TestCommand", "TestCommand description", null, new Dictionary<string, object>() { } ).Result;
+            clientMoq.Verify(c => c.CreateTaskCommandEvent("TestCommand", "", null, null), times);
 
             e = Analytics.TrackFileOperationEvent(this.TempFolder, Actions.Read, 5);
```

Comment:
How about using `TrackTaskFileOperationEvent` here?

Reply:
I did not intend to convert all obsolete APIs to the new ones with this PR.